src/client/src/rt-components/tear-off/TearOff.tsx

```diff
@@ -37,7 +37,10 @@ export default class TearOff extends React.PureComponent<Props, State> {
     if (tornOff) {
       return (
         <Portal onUnload={this.popIn} {...portalProps}>
-          <TearOffContainer>{render(this.popOut, tornOff)}</TearOffContainer>
+          <TearOffContainer>
+            <span style={{ fontWeight: 900, fontStyle: 'italic' }} />
```

Comment:
It's curious that this isn't handled here:
https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/client/public/index.html#L47